core/src/main/java/io/dekorate/kubernetes/decorator/AddRoleBindingResourceDecorator.java

```diff
@@ -54,7 +54,7 @@ public AddRoleBindingResourceDecorator(String name, String serviceAccount, Strin
   }
 
   public void visit(KubernetesListBuilder list) {
-    ObjectMeta meta = getMandatoryDeploymentMetadata(list);
+    ObjectMeta meta = getMandatoryDeploymentMetadata(list, this.name);
```

Comment:
Is this being covered by some tests? Looking at https://github.com/dekorateio/dekorate/pull/853/files#diff-d63a9fd5811d5b671a65f15670a2f2d90a61d424fb4f17073a649cdffad5f93fR390, I think this is never going to find the metadata by the name because it's using something like "name + :deployer", but maybe I'm missing something.

Reply:
What you are missing is that the suffix is `stripped`